File: app/src/main/java/com/example/msdc/api/MovieResult.java
package com.example.msdc.api;

import com.google.gson.annotations.SerializedName;

public class MovieResult {
    @SerializedName("id")
    int id;

    @SerializedName("poster_path")
    String posterPath;

    @SerializedName("title")
    String title;

    @SerializedName("vote_average")
    double voteAverage;

    public double getVoteAverage() {
        return voteAverage;
    }

    public int getId(){ return id; }

    public String getPosterPath(){ return posterPath; }

    public String getTitle(){ return title; }
}

File: app/src/main/java/com/example/msdc/api/MovieResult.kt
package com.example.msdc.api

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class MovieResult (
    @field:SerializedName("id") val id: Int? = 0,
    @field:SerializedName("poster_path") val posterPath: String? = null,
    @field:SerializedName("title") val title: String? = null
) : Parcelable